jquery.html.markdown

```diff
@@ -12,8 +12,8 @@ jQuery is a JavaScript library that helps you "do more, write less". It makes ma
 Because jQuery is a JavaScript library you should [learn JavaScript first](https://learnxinyminutes.com/docs/javascript/)
```

Comment:
`Because jQuery is a JavaScript library so, you should already be familiar with [JavaScript](https://learnxinyminutes.com/docs/javascript/)`